src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/interior_data_cache_impl.cc

```diff
@@ -1,41 +1,39 @@
 #include "rc_rpc_plugin/interior_data_cache_impl.h"
 
-
 namespace rc_rpc_plugin {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "RemoteControlModule");
 
-InteriorDataCacheImpl::InteriorDataCacheImpl() {
-
-}
+InteriorDataCacheImpl::InteriorDataCacheImpl() {}
 
-void InteriorDataCacheImpl::Add(const std::string &module_type, const smart_objects::SmartObject &module_data) {
-    LOG4CXX_AUTO_TRACE(logger_);
-    sync_primitives::AutoLock autolock(subscriptions_lock_);
-    subscriptions_[module_type] = module_data;
+void InteriorDataCacheImpl::Add(const std::string& module_type,
+                                const smart_objects::SmartObject& module_data) {
+  LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
@LuxoftAKutsan Don't we need to include logger header as well?

Reply:
@AByzhynar added 297bbe3ce2194ffbd7589ee14bbd7aad444b0e01